source/vscode/src/qirGeneration.ts

```diff
@@ -114,11 +124,40 @@ async function getQirForProgram(
     );
     if (result?.action !== "set") {
       throw new QirGenerationError(
-        error_msg +
-          " Please update the QIR target via the status bar selector or extension settings.",
+        error_msg + " Please update the QIR target via the qsharp.json.",
       );
     } else {
-      await setTarget(targetSupportsAdaptive ? "adaptive_ri" : "base");
+      // Try to update the qsharp.json manifest automatically.
```

Comment:
Was this behavior decided as part of a team discussion or something? It's weird. Can we just open the qsharp.json and let the user change the file if they want to?

Reply:
This was reflecting the existing behavior of that option, which updates the global profile setting. A button that says "Set the QIR profile to Base to continue" should do the actual setting, imho

Reply:
Changed to just open the qsharp.json to encourage the user to change the field, as suggested.